secureli/resources/files/epilog.md

```diff
@@ -11,6 +11,6 @@ SeCureLI currently supports the following languages:
 {% endfor %}
 
 
-For more information, check us out on Confluence at {{ confluence_url }}
+For more information, check us out on the GitHub wiki at {{ confluence_url }}
```

Comment:
variable would match whatever you decide it would be in setup.py

Reply:
I'm not sure what you mean by this.  Can you elaborate?